portfolio-project-matching-app/backend/dao.js

```diff
@@ -545,6 +638,49 @@ const readQuerySnapshotById = async (coll, field, id) => {
 
 }
 
+const populateAssociations = async (coll, object) => {
+    /*
+    DESCRIPTION:    populates passed object's association properties
+
+    INPUT:          coll (string): collection to get documents from
+
+                    object (object): object whose associations need populated
+
+    RETURN:         NA - alters original object passed
+    */
```

Comment:
Once again, helper function to cut down on the number of dedicated `READ` functions.